.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,86 @@
+## Type of change
+
+<!-- Mandatory
+Please delete options that are not relevant
+-->
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Enhancement
+- [ ] Breaking change
+
+## What does this PR do?
+
+<!-- Mandatory
+Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
+-->
+
+## Why is it important?
+
+<!-- Mandatory
+Explain here the WHY, or the rationale/motivation for the changes.
+-->
+
+## Checklist
+
+<!-- Mandatory
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply.
+-->
+
+- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). 
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing **unit** tests pass locally with my changes
+- [ ] New and existing **integration** tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] New dependencies have been vendorised
+
+## Author's Checklist
+
+<!-- Recommended
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+-->
+- [ ]
+
+## How to test this PR locally
+
+<!-- Recommended
+Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
+-->
+
+## Related issues
+
+<!-- Recommended
+Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.
+
+- Closes #123
+- Relates #123
+- Superseeds #123
+-->
+- 
+
+## Use cases
+
+<!-- Recommended
+Explain here the different behaviors that this PR introduces or modifies in this project, user roles, environment configuration, etc.
+
+If you are familiar with Gherkin test scenarios, we recommend its usage: https://cucumber.io/docs/gherkin/reference/
+-->
+
+## Screenshots
+
+<!-- Optional
+Add here screenshots about how the project will be changed after the PR is applied. They could be related to web pages, terminal, etc, or any other image you consider important to be shared with the team.
+-->
+
+## Logs
+
+<!-- Recommended
+Paste here output logs discovered while creating this PR, or any other output you consider important to be shared with the team.
```

Comment:
what do you mean by `logs discovered while creating this PR`?

Reply:
Stacktraces, integration logs, etc.